packages/rule-components/src/ReportDetails/index.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Grid,
+    GridItem,
+    Stack,
+    StackItem,
+    Text,
+    TextVariants,
+    Title
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import { Battery } from '@redhat-cloud-services/frontend-components/components/Battery';
+import RuleFeedback from './RuleFeedback';
+import './ReportDetails.css';
+import ReactMarkdown from 'react-markdown';
+
+const totalRiskMeta = [
```

Comment:
Can you move this to specific constant file? So we have one place where we go if we want to change something.